.github/workflows/daily-nightly-jobs.yml

```diff
@@ -394,64 +394,42 @@ jobs:
           name: ceph-upgrade-suite-quincy-artifact
           path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
-  rgw-multisite-test-with-ceph-devel:
+  canary-default:
     if: github.repository == 'rook/rook'
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        ceph-image-tag:
-          ["latest-main-devel", "latest-quincy-devel", "latest-reef-devel"]
-
     steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-
-      - name: consider debugging
-        uses: ./.github/workflows/tmate_debug
-        with:
-          use-tmate: ${{ secrets.USE_TMATE }}
-
-      - name: run RGW multisite test
-        uses: ./.github/workflows/rgw-multisite-test
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          ceph-image: quay.io/ceph/daemon-base:${{ matrix.ceph-image-tag }}
-
-      - name: upload test result
-        uses: actions/upload-artifact@v4
-        if: always()
+      - name: Invoke workflow with inputs
+        uses: benc-uk/workflow-dispatch@v1
         with:
-          name: rgw-multisite-testing-ceph-${{ matrix.ceph-image-tag }}
-          path: test
+          workflow: Another Workflow
+          # inputs: '{"ceph-image": ""}'
 
-  encryption-pvc-kms-ibm-kp:
```

Comment:
Ditto

This test is part of the canary suite that is now getting kicked off (hopefully) so I think we can remove it.